src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -0,0 +1,164 @@
+    </Style>
+
+    <Style TargetType="Grid">
+
+        <!-- GRID SPECIFIC PROPERTIES -->
+        <!--<Setter Property="RowSpacing" Value="5" />-->
+        <!--<Setter Property="ColumnSpacing" Value="5" />-->
+
+        <!-- LAYOUT INHERITED PROPERTIES -->
+        <!--<Setter Property="IsClippedToBounds" Value="False" />-->
+        <!--<Setter Property="Padding" Value="5" />-->
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource PrimaryDarkColor}}" />
+        <!--<Setter Property="Background" Value="" />-->
+        <!--<Setter Property="WidthRequest" Value="100" />-->
+        <!--<Setter Property="HeightRequest" Value="25" />-->
+        <!--<Setter Property="FlowDirection" Value="LeftToRight" />-->
+    </Style>
+
+    <Style TargetType="Image">
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource PrimaryDarkColor}}" />
+        <!--<Setter Property="Background" Value="" />-->
+        <!--<Setter Property="WidthRequest" Value="100" />-->
+        <!--<Setter Property="HeightRequest" Value="25" />-->
+        <!--<Setter Property="FlowDirection" Value="LeftToRight" />-->
+    </Style>
+
+    <Style TargetType="Label">
+
+        <!-- LABEL SPECIFIC PROPERTIES -->
+        <!--<Setter Property="HorizontalTextAlignment" Value="Left" />-->
+        <!--<Setter Property="VerticalTextAlignment" Value="Center" />-->
+        <Setter Property="TextColor" Value="{AppThemeBinding Light={StaticResource PrimaryDarkColor}, Dark={StaticResource PrimaryLightColor}}" />
+        <!--<Setter Property="CharacterSpacing" Value="0" />-->
+        <!--<Setter Property="FontFamily" Value="OpenSansRegular" />-->
```

Comment:
I think we should include a FontSize that looks good on mobile and desktop, and set the FontFamily.

If we don't have it later in the stylesheet here, I want to see different text styles for other use cases that extend this style.